src/Begin.f90

```diff
@@ -41,28 +41,28 @@ Subroutine BEGIN(param_file,spatial_file)
 !
 implicit none
 !
-    character (len=200):: param_file,source_file,spatial_file
+    character (len=200):: source_file,spatial_file
     integer:: Julian
 !
 !
 !   Mohseni parameters, if used
 !
 !
-!
-!     Card Group I
-!
+!
+!     Card Group I
+!
 read(90,*) start_date,end_date
-read(start_date,'(i4,2i2)') start_year,start_month,start_day
-read(end_date,  '(i4,2i2)') end_year,end_month,end_day
-nyear1=start_year
-nyear2=end_year
-write(*,'(2(2x,i4,2i2))')  &
- start_year,start_month,start_day,end_year,end_month,end_day
-!
-!     Establish the Julian day for which simulations begin
-!
-jul_start = Julian(start_year,start_month,start_day)
-!
+read(start_date,'(i4,2i2)') start_year,start_month,start_day
+read(end_date,  '(i4,2i2)') end_year,end_month,end_day
+nyear1=start_year
+nyear2=end_year
+write(*,'(2(2x,i4,2i2))')  &
+ start_year,start_month,start_day,end_year,end_month,end_day
+!
+!     Establish the Julian day for which simulations begin
+!
+jul_start = Julian(start_year,start_month,start_day)
+!
```

Comment:
Just make sure you use similar editor settings across the project, such as "strip whitespace at the end of the line", etc. Most editors can do this automatically. If not, we'll keep committing trivial whitespace changes, which are not informative and simply clutter the repo.